### ISSUE TITLE
QUESTION: Is this repo still maintained

### ISSUE THREAD

**Issue by Yuripetusko** [linkedin/hopscotch#284]
Hello @kate2753, @zimmi88, is this repo still maintained at all?

Thanks,
Yuri

**Comment by zimmi88** [comment]
Hi @Yuripetusko,

This repo hasn't been as actively maintained as it should be, but I'm working to ensure progress is moving ahead on Hopscotch going forward. I can't speak for @kate2753, but I'll provide my (albeit verbose) perspective on things...

A while back, I got swept up with a number of things - without going into details, the end result was that I didn't have a lot of time available to devote to Hopscotch. I should have been more communicative about this - that's an error on my part and I apologize for that.

Moving ahead, my goal is to pick up the work that was previously started to migrate Hopscotch's current functionality into a refactored ES6-based codebase that should allow us to better reason about the library's architecture and move ahead with new functionality that will make it simpler to augment Hopscotch to fit with individual use cases. This is codified as the ES6 Refresh milestone (gh72 branch); I'm unsure if this will officially be dubbed 0.3.0 or 1.0 when it's done. My preference is towards the latter simply as a means of avoiding the excuse of a pre-1.0 revision as the reason for not maintaining a solid API. But, this will depend on how confident we are in everything when all is said and done.

As a result of this, tickets that affect the current 0.2.x branch (master) are getting rolled into this milestone as focus items and things to make sure are addressed. I've noticed there's a few PRs for the current branch - if it would be helpful for the community, I'd be willing to work on a maintenance release for 0.2.x that rolls in those PRs. But, as for new features, I'll need to triage the current PRs and make an honest assessment of whether they'll actually be merged with 0.2.x or rolled into ES6 Refresh or eventually made available as a plugin. (A number of PRs may be closed without merging because of this. Authors of these PRs should feel welcome to submit new requests for these features once we're to the point that the gh72 branch is stable and merged with master and/or plugin support is added.)

Anyway, that's where we're at. Again, apologies for the verbose response, but I figured this would be a good opportunity to ensure everyone's up to date on what's up. I'm not sure what the best way to post announcements on a GitHub repo would be (issues with an "Updates" label?), so if I can figure that out I'll repost a lot of this there and/or tag this issue to improve visibility.

And, because of this, I'm going to leave this issue open for a couple of days so it doesn't disappear into the "Closed" tab. If there's any other questions or feedback about the state of the repo, please let me know! I know my track record on this project hasn't been stellar so far - I'm going to have to actively work to rebuild that feedback loop and ensure we're continuing to move forward. But do know that we want to publish the best product we can and your feedback and contributions really do help. Thanks!

=================================

In short...
- Repo maintenance had fallen by the wayside, but I'd like to get things back on track.
- Current plan is to push forward on the ES6 Refresh milestone, with preference on devoting all bandwidth to that task.
- Current PRs targeting 0.2.x will get triaged and honestly evaluated as to whether they'll be merged in or punted as an ES6 Refresh feature. This may result in a maintenance release for the 0.2.x branch.

**Comment by Yuripetusko** [comment]
Thank you for a detailed answer, really appreciate it, yes I haven't noticed gh72 branch before, so didn't see any progress. Good luck with ES6 refactor
